kinto/config/kinto.tpl

```diff
@@ -47,6 +40,29 @@ multiauth.policies = basicauth
 kinto.includes = kinto.plugins.default_bucket
 
 #
+# Auth configuration.
+#
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#authentication
+#
+kinto.userid_hmac_secret = {secret}
+multiauth.policies = basicauth
+# multiauth.policies = fxa basicauth
+
+#
+# Accounts API configuration.
+#
+# Enable built-in plugin.
+# kinto.includes = kinto.plugins.accounts
```

Comment:
I think it's maybe a bad idea to have `includes` here again, as it may conflict with the first one.

Edit: we also do it for fxa so that's probably ok.